src/main/java/nl/cherement/jak/controller/UserController.java

```diff
@@ -1,52 +1,58 @@
 package nl.cherement.jak.controller;
 
 import nl.cherement.jak.entity.UserEntity;
-import nl.cherement.jak.service.UserPrincipalDetailsService;
+import nl.cherement.jak.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.web.bind.annotation.*;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.List;
 import java.util.Optional;
 
-@CrossOrigin(origins = "*", allowedHeaders = "*")
+
 @RestController
 @RequestMapping("/user")
 public class UserController {
 
 
     @Autowired
-    private UserPrincipalDetailsService userPrincipalDetailsService;
+    private UserService userService;
+
+
+
 
 
     @GetMapping
     public List<UserEntity> findAll(HttpServletRequest request) {
-        return userPrincipalDetailsService.findAll();
+        return userService.findAll();
     }
 
     @GetMapping("/{id}")
     public Optional<UserEntity> find(HttpServletRequest request, @PathVariable("id") Long id) {
-        return userPrincipalDetailsService.findById(id);
+        return userService.findById(id);
     }
 
     @GetMapping("/me")
-    public UserEntity auth(Authentication authentication) {
-        return userPrincipalDetailsService.auth(authentication);
-    }
+    public UserEntity getUser(Authentication authentication) {
```

Comment:
Is dit nu de login endpoint? Of zit die ergens anders

Reply:
De login endpoint zit standaard in de [UsernamePasswordAuthenticationFilter](https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/web/authentication/UsernamePasswordAuthenticationFilter.html). Vandaar dat die nergens gedefineerd word.